crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -424,8 +427,11 @@ async fn c8y_mapper_syncs_pending_alarms_on_startup() {
     //     .await
     //     .unwrap();
 
+    // wait for the keep-alive time to get elapsed and the session to get closed properly by the server(broker).
```

Comment:
```suggestion
    // wait for the keep-alive time to get elapsed and the mapper's session to get closed properly by the server(broker), so that its next connection on restart is treated as a new client connection, rather than a reconnection.
```

Reply:
updated